header.yaml

```diff
@@ -0,0 +1,34 @@
+default-headers: 
+  - name: avax
+    header: |
+      // Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
```

Comment:
remove 2023 and add {YEAR}

Reply:
ah I thought we will the license headers from avax too. okay, I will resolve it.